app/typings/window.d.ts

```diff
@@ -1,11 +1,7 @@
 declare global {
-  interface NodeRequire {
+  interface ImportMeta {
     /** A special feature supported by webpack's compiler that allows you to get all matching modules starting from some base directory.  */
```

Comment:
It says "webpack", this is probably outdated. I’d update the description, but I don’t know what it’s doing. 🤡